CriteoPublisherSdk/Sources/Network/CR_CdbResponse.h

```diff
@@ -29,6 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property(copy, nonatomic, nullable) NSArray<CR_CdbBid *> *cdbBids;
 @property(assign, nonatomic) NSUInteger timeToNextCall;
+@property(assign, nonatomic, readonly) BOOL consentGiven;
```

Comment:
You added the consent flag but you didn't set it in the `.m` file.

Reply:
> LGTM for now.
> 
> And oups I didn't saw this was a draft, but I reviewed the code anyway :)

Yup sorry forgot to push this as draft directly, this is mostly to run all tests ;p